app/components/agenda/agendaitem/agendaitem-decision/component.js

```diff
@@ -52,15 +52,18 @@ export default Component.extend(isAuthenticatedMixin, {
       await this.decisionToDelete.destroyRecord();
       let agendaitemToUpdate;
 
+      const subcase = await this.agendaitem.get('subcase');
+      await subcase.get('decisions').reload();
+
       if (this.isTableRow) {
-        const subcase = await this.agendaitem.get('subcase');
-        (await subcase.get('decisions')).reload();
         agendaitemToUpdate = await this.agendaitem.content;
       } else {
         agendaitemToUpdate = await this.get('agendaitem');
       }
-      agendaitemToUpdate.set('modified', moment().utc().toDate());
       await agendaitemToUpdate.save();
+      if(!this.isDestroyed) {
```

Comment:
Can I be annoying again? Small change request: `if (` 😬

Reply:
Fixed it